### PR TITLE
feat: version viewing is now a thing

### LINKED ISSUE #3857: Publish Lifecycle: View Version
This issue is part of the larger publish lifecycle epic. 

WHEN a user selects a published version in their versions list
THEN the user should be taken to a READ ONLY route `/notebook/id/version/id` for that version

The user should then have the option to `revert` or `clone` from that point.

By `cloning` a new notebook with the versioned data should be generated and navigated to.
By `reverting`, the user should be taken to the original notebook with the data set back to that version of the notebook. This should not affect the `version` history at all

### PR DESCRIPTION
**Pull request by asalem1**
Closes #3857 

This PR creates a read only version of the version history that the user can access by clicking on the specific version they're interested in viewing.

Screen Shot 2022-02-16 at 3 47 08 PM

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 55 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3901)

File: src/alerting/components/AlertingIndex.tsx [hunk 1]
```diff
@@ -78,7 +79,10 @@ const AlertingIndex: FunctionComponent = () => {
                 <Icon glyph={IconFont.BookPencil} />
                 Now you can use Notebooks to explore your data while building an
                 alert
-                <Link to="/notebook/from/notification" onClick={recordClick}>
+                <Link
+                  to={`/${PROJECT_NAME.toLowerCase()}/from/notification`}
```
Comment (drdelambre) [inline src/alerting/components/AlertingIndex.tsx:83]: thanks for all these

Comment (asalem1) [reply src/alerting/components/AlertingIndex.tsx:83]: don't thank me. I didn't do it for tasks

**Review by drdelambre — APPROVED** [review]
seems legit

**Pull request merged**